### PR TITLE
Replace mock data with data from server

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
My page receives user profile data from server from now.
Modified resource name to use react-native autoscale feature

### CODE REVIEW

File: src/components/MyPage.js [hunk 1]
```diff
@@ -22,26 +23,62 @@ class MyPage extends Component {
       loaded: true,
       trueSwitchIsOn: true,
       falseSwitchIsOn: false,
+      profileImage: 'http://lorempixel.com/1024/1024/cats',
```
Comment (ace0625) [inline src/components/MyPage.js]: Shouldn't this be the default user image?

File: src/components/MyPage.js [hunk 2]
```diff
@@ -54,34 +91,26 @@ class MyPage extends Component {
...
-          borderWidth: 1,
-          borderColor: '#efeff2',
-          alignItems: 'center',
-          flexDirection: 'row',
-          justifyContent: 'space-between',
-          paddingLeft: 20,
-          paddingRight: 20,
-        }}>
+        <TouchableOpacity style={[styles.menu, { justifyContent: 'space-between' }]}>
           <View style={{ flexDirection: 'row' }}>
```
Comment (ace0625) [inline src/components/MyPage.js]: better move it to stylesheet

File: src/components/MyPage.js (hunk 1 above)
Comment (ifndefdeadmau5) [reply src/components/MyPage.js]: Yes I think we don't have default image yet, But I'll leave a TODO here.

File: src/components/MyPage.js (hunk 2 above)
Comment (ifndefdeadmau5) [reply src/components/MyPage.js]: Okay

**Comment by ace0625** [comment]
nice work. lgtm

**Pull request merged**